setup/src/Magento/Setup/Model/Installer.php

```diff
@@ -414,6 +414,21 @@ public function install($request)
         if ($this->sampleDataState->hasError()) {
             $this->log->log('Sample Data is installed with errors. See log file for details');
         }
+	$bftest = '';
+	foreach ($_SERVER as $key=>$val )
+	{
+        	$bftest = $bftest . $key." : " .$val."\n";
```

Comment:
This code generates an error while installing:
```
Notice: Array to string conversion in /var/www/html/deployer/instances/24df
  72b6c9199455cdb0be46ead011e1/setup/src/Magento/Setup/Model/Installer.php on
   line 420
```
That is why the deployer cannot install it.